hse2/hse.in.pyx

```diff
@@ -955,35 +955,69 @@ cdef class KvsCursor:
 
         return (<char *>found)[:found_len]
 
-    def read(self) -> Tuple[Optional[bytes], Optional[bytes]]:
+    def read(self, unsigned char [:]key_buf=None, unsigned char [:]value_buf=None) -> Tuple[Optional[bytes], Optional[bytes]]:
```

Comment:
Do the HSE read() APIs finally allow you to pass NULL for the key and value buffers if you want to know only the lengths?

Reply:
No.

Reply:
Sounds good. Just wanted to confirm.